src/app/modules/group/components/joined-group-list/joined-group-list.component.ts

```diff
@@ -55,6 +55,7 @@ export class JoinedGroupListComponent implements OnDestroy {
     const groupId = group.group.id;
     const groupName = group.group.name;
     this.joinedGroupsService.leave(groupId)
+      .pipe(first())
```

Comment:
I don't get the point of this. All http services only emit once and then complete.

Reply:
Yes, l got TS notification from IDE but right now it's gone, strange... I'll remove then

Reply:
Fixed